net/strongswan/files/ipsec.init

```diff
@@ -166,9 +166,9 @@ config_conn() {
 	config_get reqid                    "$1"           reqid
 	config_get packet_marker            "$1"           packet_marker
 
-	[ -n "$local_nat" ] && local_subnet=$local_nat
+	[ -n "$local_nat" ] && local_subnet="$local_nat"
 
-	ipsec_xappend "conn $config_name-$1"
+	ipsec_xappend "conn $1"
```

Comment:
I don't see the point to prefixing the section with 'cfgNNNN-' so let's just keep the same name.